source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -35,35 +42,80 @@ partitions = this would theoretically provide you up to: ``~896,000
 PUT/POST/DELETE TPS and 1,408,000`` GET TPS if requests are distributed evenly
 across the partitions).
 
+Configuration
+^^^^^^^^^^^^^
+
 .. code-block:: none
 
   plugin {
-    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
-    obox_index_fs = compress:gz:6:s3: https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_fs = fscache:1G:/var/cache/mails:compress:gz:6:aws-s3:https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
+    obox_index_fs = compress:gz:6:aws-s3:https://ACCESSKEY:SECRET@BUCKETNAME.s3.amazonaws.com/
   }
 
+.. Note::
+        If the version you are using does not support the ``aws-s3`` scheme,
+        try using the more generic ``s3`` scheme.
+
+        .. versionadded:: 2.3.10
+
 Get ACCESSKEY and SECRET from `www.aws.amazon.com <https://aws.amazon.com/>`_
 ``-> My account -> Security credentials -> Access`` credentials. Create the
 ``BUCKETNAME`` from ``AWS Management Console -> S3 -> Create Bucket``.
 
 If the ``ACCESSKEY`` or ``SECRET`` contains any special characters, they can be
 %hex-encoded.
 
+If the ``aws-s3`` scheme is used Dovecot defaults to prepend the following URL
```

Comment:
Is it really "prepend"?  Or does it append?

Reply:
It's prepend now (it changed), we thereby allow users to override defaults.

Reply:
(Reminder: Don't click unresolve conversation before clicking comment, or it will lose the comment.)

I wonder where Markus's comment about this went that I see in my email:

> It's prepend now (it changed), we thereby allow users to override defaults.

Anyway now that we removed auth_protocol setting (-> remove it from these changes), there's no longer any difference between prepend and append since the "addhdrvar" and "loghdr" can't be overridden. But I suppose this could still be kept as "prepended"..